src/main/java/docoverride/eventbus/Examples.java

```diff
@@ -38,14 +41,29 @@ public void example11(EventBus eventBus, MessageCodec myCodec) {
     eventBus.send("orders", new MyPOJO());
   }
 
+  public void example12(EventBus eventBus) {
+    // Class does not implement Shareable
+    eventBus.registerDefaultCodec(MyPOJO.class, MessageCodec.localCodec(MyPOJO.class, MyPOJO::copy));
+
+    // Class implements Shareable
+    eventBus.registerDefaultCodec(MyPOJO.class, MessageCodec.localCodec(MyPOJO.class));
```

Comment:
If it's `Shareable` wouldn't it be enough just to say:

```java
eventBus.registerDefaultCodec(MyPOJO.class);
```

It feels a bit redundant to have to declare a codec.